client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
I don’t know to be honest. It’s already a bit of a mess. Why do both `quiet` and `noInfo` exist?